Activity.js

```diff
@@ -45,20 +45,21 @@ class Activity {
 
   markComplete() {
     this.completed = true;
-    pastActivities.push(this);
-    console.log(pastActivities);
+    // pastActivities.push(this);
+    // console.log(pastActivities);
   }
 
   saveToStorage() {
```

Comment:
**Suggestion:** If y'all are writing tests then `saveToStorage` and `getFromStorage` are two excellent things to test.